index.js

```diff
@@ -1,32 +1,33 @@
-module.exports = function (handlebars) {
-  'use strict';
+'use strict';
 
-  handlebars = handlebars || require('handlebars');
+var fs = require('fs');
+var path = require('path');
+var options = { 'encoding': 'utf8' };
+
+function createId () {
+  return (10e8 * Math.random()).toString(16);
```

Comment:
generates strings like `2a160dd8.211922` .. so `$('2a160dd8.211922')` will not find these nodes